### ISSUE TITLE
Clear UI Bookmark as Cache Type

### ISSUE THREAD

**Issue by PivitParkour94** [magento/community-features#215]
It was hinting to in #4620 but because it isn't a specific issue related to the `ui_bookmark` not being cleared, I thought it might be worth creating an issue here. Thinking about it a little more, it seems like it might be more appropriate to be raised as a feature request to have a custom cache type for the ui_bookmarks. We could even go so far as to have a grid where admin users can remove specific entries in the ui_bookmark table as a UI component grid (but the layout might get stuck in an uneditable state, how ironic).

Is this something that we can have? If it's approved I wouldn't mind doing the development for it if that is required. 

Ultimately, it feels like the current solution to refresh the layout of a custom ui_component listing, which requires clearing the row in the `ui_bookmark` table is too technical for a store owner or a QA user for example.

Please let me know whether this has any hang ups before the development effort is made in vain.

**Comment by PivitParkour94** [comment]
Is there any further hold ups to this being done?

**Comment by fredden** [comment]
We have installed this module on our websites to allow admin users to reset their own UI bookmark cache.
https://packagist.org/packages/element119/module-reset-bookmark-ui

**Comment by PivitParkour94** [comment]
@fredden that module looks like a good workaround, but this issue was really to have Magento address the issue in the core product. It's been 3 years and no real communication has been made, so I'll just give up trying to suggest changes (I'm not even using Magento in dev anymore)

**Comment by thomas-kl1** [comment]
@PivitParkour94 please don't close this unless it's shipped in Magento core

**Comment by fredden** [comment]
I agree; let's keep this open until it's actually resolved in the core product. Having a third-party work-around is great in the meantime, but doesn't solve the problem that is described in this feature request.

**Comment by PivitParkour94** [comment]
Agree, I have just lost hope that Magento will do anything about it. If anything me closing this just demonstrates that. I no longer make any issues on the magento github because they just get brushed off. It's disappointing

**Comment by thomas-kl1** [comment]
@PivitParkour94 also the issue is tagged for 2.5, however Adobe didn't planned to release 2.5 in 2023 😢
